api/routes/api_routes.py

```diff
@@ -145,11 +141,15 @@ def get_median_home_value(id):
 @api.route('/api/v1/median-home-value', methods=['GET'])
 def all_median_home_value():
     return medianhomevaluecontroller.all_median_home_value()
-
-@api.route('/api/v1/median-home-value/<id>', methods=['PUT'])
-def update_median_home_value(id):
-    return medianhomevaluecontroller.update_median_home_value(id)
-
-@api.route('/api/v1/median-home-value/<id>', methods=['DELETE'])
-def destroy_median_home_value(id):
-    return medianhomevaluecontroller.destroy_median_home_value(id)
+#
+# @api.route('/api/v1/median-home-value', methods=['POST'])
```

Comment:
I see we are commenting these routes until Iteration2 - where we have admin authorization on these routes